### DISCUSSION TITLE
the network connection disappears and never reappears.

### DISCUSSION THREAD

**Discussion by satiromarra** [utmapp/UTM#4540] [UTM Help (macOS)] [answered]
**Describe the issue**  
When I start the VM the network connection disappears and does not appear again, I have tried all the configurations without result.

The problem has already happened to me several times and I end up reinstalling a new VM.
<img width="455" alt="image" src="https://user-images.githubusercontent.com/1863162/197207045-d5872fe4-b649-4ab3-8b39-c1eae0bb692d.png">

**Configuration**  
* UTM Version: 4.0.8 & 3.x
* macOS Version: 12.6
* Mac Chip (Intel, M1, ...):  M1

**Debug log**  
[debug.log](https://github.com/utmapp/UTM/files/9839371/debug.log)

**Upload VM**  
[config.zip](https://github.com/utmapp/UTM/files/9839348/config.zip)

**Comment by osy** [comment, marked as answer]
Check that the network adapter name did not change. https://docs.getutm.app/guides/ubuntu/#networking-is-unavailable Is for ubuntu but it could be a similar issue

**Reply by satiromarra** [reply]
Thanks, it worked!

I had set enp0s6 instead of enp0s1
How is it possible to change the name of the network?

**Reply by osy** [reply]
I'm not sure what distro you're using. You can probably adapt the ubuntu instructions. If not, you can ask on the UTM discord and see if someone can help you.

**Reply by satiromarra** [reply]
I currently have Kubuntu (Ubuntu) but something similar happened to me with a Debian.
